Automattic/simplenote-macos
# Fixing Emojis Crash

### Fix
In this PR we're mapping a (patched) version of Simperium, which makes sure `Surrogate Pairs` aren't split in the middle, and thus, causing invalid unicode strings.

@danielebogo May I bug you with a real fun PR?
Thanks in advance!!

Closes #397

### Test
1. Add a new note
2. Enter the following emojis: ☺️🖖🏿
3. Insert the following emoji in between: 😃

- [x] Verify the app doesn't break!

### Release
`RELEASE-NOTES.txt` was updated in 3af363e with:
 
> Fixed a bug that caused certain emojis combinations to crash the app
